plugin/src/test/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfiguratorTest.java

```diff
@@ -90,6 +92,25 @@ public void export_mapping_should_not_be_null() throws Exception {
         assertEquals("plainText", configNode.getScalarValue("markupFormatter"));
     }
 
+    @Test
+    public void shouldThrowConfiguratorException() {
+        Mapping config = new Mapping();
+        config.put("foo", "foo");
+        config.put("bar", "abcd");
+        config.put("qix", "99");
+        ConfiguratorRegistry registry = ConfiguratorRegistry.get();
+        try {
+            registry.lookupOrFail(Foo.class).configure(config, new ConfigurationContext(registry));
+            fail("above action is excepted to throw ConfiguratorException!");
```

Comment:
When I ran the test I directly hit the fail in this case without the configuratorException being thrown.